upstream-community-operators/gpu-operator/1.0.0/gpu-operator.v1.0.0.clusterserviceversion.yaml

```
@@ -0,0 +1,207 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"clusterpolicies.nvidia.com/v1","kind":"ClusterPolicy","metadata":{"name":"nvidia-gpu-example"},"spec":{"size":3},"status":{"state":"N/A"}}]'
+    capabilities: 'Basic Install'
+    categories: "AI/Machine Learning, Integration & Delivery, Monitoring, Cloud Provider, OpenShift Optional"
```

```suggestion
    categories: "AI/Machine Learning, OpenShift Optional"
```

I don't think this Operator fits `Monitoring` or `Cloud` as categories.